src/Algorithms/Abstraction.php

```diff
@@ -69,11 +67,11 @@ public function __construct( $points = null )
      *
      * @param mixed $points Must be array or Traversable
      *
-     * @return null
```

Comment:
Totally weird. I'm completely indifferent if return should be set (in this case). BUT I like to do whatever the code document / lint program suggest. For this project, it's php's code_sniffer.

Please run `vendor/bin/phpcs src` on this branch, and resolve any issues emitted. Happy to accept this MR when both phpcs  + phpunit passes.